src/lib/updateVersion.js

```diff
@@ -0,0 +1,23 @@
+const fs = require("fs");
+const path = require("path");
+const getVersion = require("./getVersion.js");
+const chalk = require("chalk");
+
+module.exports = function updateLastVersionUsed() {
+  // Check version in .gluestick file
+  const gluestickDotFile = path.join(process.cwd(), ".gluestick");
+  var fileContents = fs.readFileSync(gluestickDotFile, {encoding: "utf8"})
+  // We won't know how old this version is, it might have still have the 'DO NOT MODIFY' header
+  fileContents = fileContents.replace("DO NOT MODIFY", "");
```

Comment:
I thought we decided not to do the whole `DO NOT MODIFY` line on the top. The reason is, it would be nice if scripts could just parse this file as JSON without having to strip the header. What do you think?